site/composer.json

```diff
@@ -38,7 +38,8 @@
 		"php-http/guzzle6-adapter": "^1.1",
 		"symfony/cache": "^3.2",
 		"javiereguiluz/easyadmin-bundle": "^1.16",
-		"beberlei/DoctrineExtensions": "^1.0"
+		"beberlei/DoctrineExtensions": "^1.0",
+		"sentry/sentry-symfony": "dev-master"
```

Comment:
I think we should stick with the 0.8.x series for now. 1.0 may have some issues

Reply:
thanks, I will rebase with the correct version.